ansible/tasks/logging/README.md

```diff
@@ -0,0 +1,34 @@
+# Playbooks to install FluentD and FluentD plugins
+
+These playbooks install FluentD on Windows instance in order to enable logging support.
+
+Before running these playbooks,
+- Set up an OpenShift cluster
+- Ensure Ansible environment is set up to run the playbook. For a Fedora user, sample steps are:
+```
+$ sudo dnf install libselinux-python
+$ pip install ansible pywinrm
+```
+- A Windows Server 2019 running in the same VPC as the OpenShift cluster with winrm enabled.
+You can use `wni` tool to setup the Windows node following this [link](https://github.com/openshift/windows-machine-config-operator/tree/master/tools/windows-node-installer).
+The winrm module is enabled automatically if you use `wni` tool to create the server node.
+- Login to your cluster using `oc` command.
+
+Modify the `hosts` file with the instance information.
```

Comment:
Can you overwrite the variables that you're having in the hosts file from command line?

Reply:
We can, however my problem mentioned in [comment above](https://github.com/openshift/windows-machine-config-operator/pull/42#discussion_r331263496) brings up the issue of whether we can make version of ruby installed dynamic. Since `creates_path` is a mandatory field, with version dependency, I am not sure if we should let the user decide the version of ruby installed

Reply:
Yes, that can be done. Supplying from command line can also override the existing variable in the file